src/runtime/internal/utils.ts

```diff
@@ -66,6 +66,9 @@ export function validate_store(store, name) {
 
 export function subscribe(store, ...callbacks) {
 	if (store == null) {
+		for (const callback of callbacks) {
+			callback(store);
```

Comment:
1st argument of `callback` is a value of store, so in this case, the value is undefined right?
value of null/undefined is undefined.

```suggestion
			callback(undefined);
```

Reply:
But `null !== undefined`. This is returning the same value passed, and preserving runtime correctness as checked by TypeScript.

Reply:
Regarding `store.value`,
if `store` is null, `store.value` = `null.value`. `value` is undefined.
if `store` is undefined, `store.value` = `undefined.value`. `value` is undefined.

https://svelte.dev/docs#component-format-script-4-prefix-stores-with-$-to-access-their-values-store-contract

Reply:
I don't see an implicit conversion of `null` into `undefined` in the linked section, but my reading skills this morning aside - it seems to match the runtime behavior that exists today (minus this bug) 🙂  [REPL confirming `null -> undefined`](https://svelte.dev/repl/2ba45e46a5b74f22bec78c6a4aa670be?version=3.53.1)

If this is the desired behavior, the types for TypeScript need to be fixed too. This code right now fails to compile:

```
    let nameStore = null;

    let name: undefined;
    $: name = $nameStore;
```
`Type 'null' is not assignable to type 'undefined'.`

That's why I wrote the PR the way I did.